src/meta/src/barrier/mod.rs

```diff
@@ -766,10 +766,10 @@ impl GlobalBarrierManager {
 
         send_latency_timer.observe_duration();
 
-        let node_to_collect = match self
-            .control_stream_manager
-            .inject_barrier(command_ctx.clone())
-        {
+        let node_to_collect = match self.control_stream_manager.inject_barrier(
+            command_ctx.clone(),
+            self.state.inflight_actor_infos.existing_table_ids(),
```

Comment:
Just to confirm, `self.state.inflight_actor_infos.existing_table_ids()` contains both created MV's and creating MV's state table ids, right?

Reply:
Yes. This is more like a refactoring PR, and the logic of partial checkpoint based backfill is not included yet.